Accessibility docs page tweaks

junpataleta opened the pull request:
- Make the wording more generic and not just pertaining to a single occasion of the accessibility audit.
- Remove the accreditation badges (especially since they are currently expired) and replace them with links to the accreditation pages.
- List down the dates when WCAG 2.1 Level AA accreditation was issued on the given Moodle versions on a table.

<a href="https://gitpod.io/#https://github.com/moodle/devdocs/pull/513"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

andrewnicols reviewed (approved):
Is it worth adding a link to the relevant issue for each of the WCAG reviews?

Also is it worth noting that the accreditation does expire, but that we still achieved compliance in the first place and that we do not recertify versions after their initial certification. That seems to be the number two recurring question after "When is Version X going to be certified?"

I'll let you self-merge this if you want to now, or you can add a patch with any of the above here or in a new issue. Up to you.

junpataleta commented:
Thanks, @andrewnicols. I added a [fixup commit](https://github.com/moodle/devdocs/pull/513/commits/a5d2bf96859d42c421b2115fb2b1d58f32e3331b) where:
- The epic links for the accessibility audits are added on the table for Moodle LMS
- I added a note about the expiry of the accreditation. Though I am not sure if I worded it appropriately or whether it can be further improved/made clear. Please feel free to suggest improvements. 

Marie will also be checking this out sometime soon and am looking for her feedback too.

Cheers!

andrewnicols commented on general/development/policies/accessibility.md:
```diff
@@ -16,18 +16,18 @@ Key pages within Moodle, which are representative of its overall accessibility a
+Both Moodle LMS and the Moodle Mobile App hold a WCAG 2.1 AA accreditation based on the successful resolution of issues identified during audits.
 
-These accreditations are renewed each year.
+The accreditations issued do expire after a year. Though it does not mean that the Moodle versions that were accredited are no longer accessible nor they are no longer supported for accessibility. It just means that the process for the accreditation of the latest supported Moodle version(s) are still in progress or pending.
```
I'd suggest something like t his maybe:

```suggestion
Accreditations expire after one year, but this does not mean that the accreditation is invalid or that the Moodle version is no longer accessible. An expired accreditation just means that the process for re-accrediting the latest supported version or versions of Moodle is currently in progress.
```

Remove the distinction between "in progress" and "pending", and put it into a slightly more complete and full sentence.

junpataleta replied:
Thanks, Andrew! I have applied your suggestion, amended the fixup commit and squashed it to the original commit.
